### ISSUE TITLE
SVG export file generates errors in Firefox

### ISSUE THREAD

**Issue by Xermit** [jgraph/drawio-desktop#611]
Hi,
I'm using the desktop version of diagrams.net, rev. 14.5.1, on W7.
My problem is the following:
I export a diagram as svg file. When I then open this file in Firefox v88, the diagram is not displayed. Instead, I get an XML Parsing Error (in variations: unclosed token, not well-formed).
This occurs even with the simplest diagram, containing a rectangular shape with no text, fill or whatever.

Strangely enough, in recent past this mechanism did work correct with more complicated diagrams, in the same set-up. These diagrams still behave as expected in Firefox, only new diagrams generate this error.
In IE11, all diagrams are displayed correct.

Any suggestions where to look for a solution?
Thx, Siebrand

**Comment by davidjgraph** [comment]
Could you update to the latest desktop build and test again please?

**Comment by Xermit** [comment]
I updated to the latest drawio desktop build (14.6.13), updated Firefox to 89.0, and cleared the browser cache after each test.
Some diagrams now display correct. Others still don't.
I seems that changing anything in the layers of a drawing (be it the renaming of the default layer, or adding a new layer) triggers the errors. But this may also be a coincidence.
Should this be of any help, I have attached one diagram (in drawio and svg format) that, at least in my setup, generates the mentioned error.
[ExportTest.Layers.zip](https://github.com/jgraph/drawio-desktop/files/6582750/ExportTest.Layers.zip)

**Comment by Matheus-Garbelini** [comment]
Hi, is it possible to export SVG with texts transformed into paths?
I'm having some issues with some unknown fonts being used in the SVG

**Comment by Xermit** [comment]
I found the cause of my problem: The svg files as read by Firefox were corrupted, and some were not even updated when I modified the source file. This pointed me towards my local server, that runs in a virtual machine on my desktop, and that serves the svg files. After resetting this virtual machine, the errors have disappeared.
What is wrong with my virtual machine is still to be found out, but this is not a topic for this forum.
